packages/store/src/state-factory.ts

```diff
@@ -64,7 +89,7 @@ export class StateFactory {
 
       const instance = this._injector.get(klass);
 
-      mappedStores.push({
+      mappedStores.push(<MappedStore>{
```

Comment:
Wouldn't the type be implied based on the array?

Reply:
The explicit typing ensures that any deviation from the interface gets caught by the typing system. I'll double check, but I think that it is necessary.

Reply:
If mappedStore is type correctly I don't think you need it